hack/make/test-integration-flaky

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -e -o pipefail
+
+source hack/validate/.validate
+new_tests=$(
+    validate_diff --diff-filter=ACMR --unified=0 -- 'integration/*_test.go' |
+    grep -E '^(\+func )(.*)(\*testing)' || true
+)
+
+if [ -z "$new_tests" ]; then
+    echo 'No new tests added to integration.'
+    return
+fi
+
+echo
+echo "Found new integrations tests:"
+echo "$new_tests"
+echo "Running stress test for them."
+
+(
+    TESTARRAY=$(echo "$new_tests" | sed 's/+func //' | awk -F'\\(' '{print $1}' | tr '\n' '|')
+    export TESTFLAGS="-test.count 5 -test.run ${TESTARRAY%?}"
+    export TEST_REPEAT=5
```

Comment:
@olljanat @thaJeztah  why set both `-test.count 5` and `TEST_REPEAT 5` ? That will run 25 times. Also, if you do set `-test.count` you need to `export TIMEOUT=...` (which will set `-test.timeout`) to 5 times the existing TIMEOUT.

Reply:
5 runs is not enough to find example known flaky test TestCreateServiceSecretFileMode. Look comments above when I tested to run this to existing tests.

Difference between this and just run 25 times is this will restart daemon after every 5 test runs. 

Increasing timeout of course can be done if needed.

Btw. Did you saw this timeouting on some PR now?

Reply:
@olljanat yes on our internal EE fork. I'm fixing the bug in https://github.com/moby/moby/pull/38693